src/components/Summary/SummaryItem.js

```diff
@@ -13,8 +13,8 @@ import { scroller } from 'react-scroll';
 import summaryStyles from './summaryStyles';
 import { createShortName } from './utils';
 
-function SummaryItem({ definition, request, renderSummary, subText }) {
-  const classes = summaryStyles();
+function SummaryItem({ definition, request, renderSummary, subText, color }) {
+  const classes = summaryStyles({color});
```

Comment:
The color field is defined under index definition object, and since the definition object is already being passed into SummaryItem, we can access color by doing: const classes = summaryStyles({definition.color}). This will avoid to pass an extra props. Everything else look good to me

Reply:
Good catch !